lib/packages/core/README.md

```diff
@@ -16,6 +20,20 @@ or using [yarn](https://yarnpkg.com/):
 yarn add @burstjs/core
 ```
 
+### Using in classic `<script>`
+
+
```

Comment:
Missing bit about `<script>` tag maybe?

Reply:
I need to be sure which link will be. I intend to update the readme in an upcoming PR